Rovers/Rovers.tex

```diff
@@ -0,0 +1,344 @@
+
+\subsubsection{Rover Hardware}
+\label{roverhardware}
+The rover will need to be constructed and tested. Because of this a physically constructed rover is needed. This will be necessary for ImagineRIT for cool factor. This includes the rover body, mast, suspension, and wheels. 
+
+\begin{figure}[ht!]
+  \includegraphics[width=\linewidth]{figs/rocker-bogie.png}
+  \caption{Rocker-Bogie mechanism}
+\label{fig:lifecycle}
+\end{figure}
+
+\subsection{Milestones}
+\label{subsec:milestones}
+The largest milestones for the technical report would be the would be draft and completion. 
+The rover itself would include the various CAD models of the rover, software milestones, electrical system completion, and drive train system. 
+Other milestones include securing funding and completion of the learning period at the beginning of the fall semester.
+
+\begin{table}[ht!]
+    \caption{Timeline}
+    \centering
+    {\renewcommand{\arraystretch}{1.5}
+    \begin{tabularx}{\linewidth}{LL} 
+    \hline
+    \textbf{Time} & \textbf{Phase} \\
+    \hline
+    August & Learning and Planning Phase \\
+    September - November & Design Phase \\
+    December - February & Manufacturing Phase \\
+    March - May & Testing Phase \\
+    \hline
+    \end{tabularx}
+    }
+\label{tab:timeline}
+\end{table}
+
+\section{Externalities}
+\subsection{Prerequisite Skills}
+The project is going to require members that are knowledgeable in mechanical design and manufacturing, software engineering, (potentially) machine learning, electrical engineering, and project management. These members will also need to be experienced in research and design document work. Though this seems simple, finding SPEX members to do this well may be a challenge due to competition from other projects. 
+
+Building a rover, especially one built to URC specifications requires a large team with a diverse set of technical skills. 
+This project will require SPEX to look at and asses what areas are sufficient and what areas need to be developed in order to build a rover. 
+It will also let us know to what spec we are currently capable of building such a rover. 
+
+\subsection{Funding Requirements}
+The rover project is looking at a minimal funding amount of \$500.00 USD from RIT SPEX. This funding is needed to make a rover that fulfills the MVP defined in the Primary Objective section. In order to give a better deliverable the team will also be reaching out to a large list of companies looking for further funding through sponsorship. Increasing the funding to \$2500.00 USD would allow for the team to construct a more feature complete rover. Components like LIDAR sensors and long-range remote communications are expensive. Though a rover can be made without them additional funding allows for the rover project to expand in scope. 
```

Comment:
Does the $500 leave room for any improvements? Do some improvements cost more than others?

Reply:
Some software inprovements such as partial autonomy. The battries might also be a possibility.